kornia/feature/integrated.py

```diff
@@ -173,6 +174,37 @@ def __init__(self,
         super().__init__(detector, descriptor)
 
 
+class KeyNetHardNet(LocalFeature):
+    """Convenience module, which implements KeyNet detector + HardNet descriptor."""
+    def __init__(self,
+                 num_features: int = 8000,
+                 upright: bool = False,
+                 device: torch.device = torch.device('cpu')):
```

Comment:
do we need device here ? in principle, IF `LocalFeature` inherits from Module and we properly register stuff, it shouldn't be needed ELSE add a todo note